ftw/mobile/templates/navigation.html

```diff
@@ -23,8 +23,11 @@
                          title="{{i18n "label_goto_parent"}} {{parentNode.title}}">
                           <span>{{i18n "label_goto_parent"}} {{parentNode.title}}</span>
                       </a>
-
-                      <a href="{{parentNode.url}}">{{parentNode.title}}</a>
+                      {{#if parentNode.title}}
+                        <a href="{{parentNode.url}}">{{parentNode.title}}</a>
+                      {{else}}
+                        <a href="{{parentNode.url}}">Startseite</a>
```

Comment:
I just implemented this feature for experimental use. The customer has first to decide if this feature is okay for him. Then it's absolutely clear to have a translated string.